docs/documentation/server_admin/topics/authentication/authentication-sessions.adoc

```diff
@@ -0,0 +1,25 @@
+
+[[_authentication-sessions]]
+=== Authentication sessions
+
+When a login page is opened for the first time in a web browser, {project_name} creates an object called authentication session that stores some useful information about the request.
+Whenever a new login page is opened from a different tab in the same browser, {project_name} creates a new record called authentication sub-session that is stored within the authentication session.
+Authentication requests can come from any type of clients such as the Admin CLI. In that case, a new authentication session is also created with one authentication sub-session.
+Please note that authentication sessions can be created also in other ways than using a browser flow.
+
+The authentication session usually expires after 30 minutes by default. The exact time is specified by *Login timeout* switch in the *Sessions* tab of the admin console when <<configuring-realms,configuring realm>>.
+
+==== Authentication in more browser tabs
+
+As pointed above, it can happen that user is trying to authenticate to {project_name} server from multiple browser tabs of same browser. However when user authenticates in one of browser tabs,
+the other browser tabs will automatically restart the authentication in themselves. This is done due the small javascript available on {project_name} login pages. The restart will typically
+automatically authenticate user in other browser tabs and redirect to clients because there is SSO session now due the fact that user just successfully authenticated in first browser tab. There
```

Comment:
```suggestion
authenticate the user in other browser tabs and redirect to clients because there is an SSO session now due to the fact that the user just successfully authenticated in first browser tab. 
```